test/DynamoCoreTests/DSEvaluationViewModelTest.cs

```diff
@@ -1098,6 +1098,13 @@ public void List_Map_Default_5233()
         }
 
         [Test]
+        public void TestListCombineRegress5561()
```

Comment:
Im guessing there was no test that caught the regression before and this test would be unique to this bug.